docs: remove unnecessary step in tutorial

FrederikSchlemmer opened the pull request:
closes #25025 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: #25025 


## What is the new behavior?

The unnecessary step of clicking the "go back" button has been deleted. To avoid any confusion about the following navigation to the previous view after saving the change the bahaviour is explained.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

mary-poppins commented:
You can preview bff8bfa at https://pr25059-bff8bfa.ngbuilds.io/.

brandonroberts commented on aio/content/tutorial/toh-pt6.md:
```diff
@@ -294,8 +294,7 @@ That header is in the `httpOptions` constant defined in the `HeroService`.
 
-Refresh the browser, change a hero name, save your change,
-and click the "go back" button. 
+Refresh the browser, change a hero name and save your change. The following navigation to the previous view is implemented in the `save()` method defined in `HeroDetailComponent`.
```
Change to `Navigating to the previous view ...`. "The following" makes it seem like there is code below this text.

FrederikSchlemmer replied:
Thanks for the feedback. I changed this wording.

brandonroberts reviewed (changes requested):
Squash commits and update commit message to `docs: remove unnecessary step in tutorial for HTTP`

brandonroberts commented:
Thanks for the PR @FrederikSchlemmer! I left you some feedback.

mary-poppins commented:
You can preview 2464fc3 at https://pr25059-2464fc3.ngbuilds.io/.

mary-poppins commented:
You can preview 4adb10c at https://pr25059-4adb10c.ngbuilds.io/.

FrederikSchlemmer commented:
@brandonroberts I squashed the new commits and renamed it. 

I'm not sure why the update commit of my fork is integrated in the pull request. If I squash this too, the update is counted with the files changed, so that the number increases to 345.

What's the best way to proceed?

brandonroberts commented:
The best way is to rebase on master in your fork

1. Checkout and pull down latest `master` from upstream (git checkout master) (git pull upstream master)
2. Rebase your branch on top of master with the updated commit message (git checkout TourOfHeroesCorrection) (git rebase master -i)
3. Push your branch back up (git push origin --force-with-lease TourOfHeroesCorrection)

mary-poppins commented:
You can preview b51d7f6 at https://pr25059-b51d7f6.ngbuilds.io/.

mary-poppins commented:
You can preview 4e6002a at https://pr25059-4e6002a.ngbuilds.io/.

mary-poppins commented:
You can preview 9d02c9f at https://pr25059-9d02c9f.ngbuilds.io/.

FrederikSchlemmer committed:
docs: Change unnecessary step in ToH-Tutorial

FrederikSchlemmer commented:
@brandonroberts Thanks for the instructions! Everything is squashed in one commit with the updated commit message.

mary-poppins commented:
You can preview b50365d at https://pr25059-b50365d.ngbuilds.io/.
